snap7/six.py

```diff
@@ -165,10 +165,10 @@ def __init__(self, six_module_name):
 
     def _add_module(self, mod, *fullnames):
         for fullname in fullnames:
-            self.known_modules[self.name + "." + fullname] = mod
+            self.known_modules[f"{self.name}.{fullname}"] = mod
```

Comment:
if you drop python2  support you can drop the whole six.py module.

Reply:
I would do this in another PR, so that this PR is focused on the f-strings.

Reply:
but you already disable the py2 tests in this PR

Reply:
Is removed.